styles/layouts/Error.module.scss

```diff
@@ -0,0 +1,29 @@
+.container {
+    @apply flex flex-col;
+    @apply size-full h-screen;
+}
+
+.page {
+    @apply flex flex-1;
+    @apply rounded-t-lg p-4;
```

Comment:
if we round the top, are we not gonna just have some whitespace at the very top of it? (when we get the new navbar)